src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -55,42 +53,42 @@ function LtiConfigForm({
             }}
           />
         </p>
-        <Form.Group controlId="consumerKey" isInvalid={isInvalidConsumerKey}>
-          <Form.Label>{intl.formatMessage(messages.consumerKey)}</Form.Label>
+        <Form.Group controlId="consumerKey" isInvalid={isInvalidConsumerKey} className="mb-4">
           <Form.Control
+            floatingLabel={intl.formatMessage(messages.consumerKey)}
             onChange={handleChange}
             onBlur={handleBlur}
             value={values.consumerKey}
           />
           {isInvalidConsumerKey && (
-          <Form.Control.Feedback type="invalid">
-            {errors.consumerKey}
+          <Form.Control.Feedback type="invalid" hasIcon={false}>
+            <small>{errors.consumerKey}</small>
```

Comment:
is this something which is suggested in figma?

Reply:
figma has 12px font style for error messages but we have default font style is 15px so by this way we can achieve most closest design. small font style is 13.5px.